devices/generic/subdevices/extended_color_light.json

```diff
@@ -6,8 +6,6 @@
     "order": 10,
     "uuid": ["$address.ext", "0x01"],
     "items": [
```

Comment:
`state/reachable` should be mandatory.

Reply:
`state/reachable` is already in the items list